src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_upscale.py

```diff
@@ -90,6 +91,25 @@ def __init__(
     ):
         super().__init__()
 
+        # check if vae has a config attribute `scaling_factor` and if it is set to 0.08333, else set it to 0.08333 and deprecate
+        is_vae_scaling_factor_set_to_0_08333 = (
+            hasattr(vae.config, "scaling_factor") and vae.config.scaling_factor == 0.08333
```

Comment:
this is a bit dangerous with `== and float numbers no?`

I would maybe just check if the vae has a scaling factor and if it does, we don't deprecate anything.

Reply:
Can you please also open a PR for the upscaling pipeline to make sure it's updated there?

Reply:
I mean open a PR here: https://huggingface.co/stabilityai/stable-diffusion-x4-upscaler/discussions